### PR TITLE
DOCS-12470 Salesforce CDP 1.1

### PR DESCRIPTION
**Pull request by sathya0**

### CODE REVIEW

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,1163 @@
...
+** <<Insights - Get Insights API>> +
+Shows how to query the calculated insights. Users can slice, dice, and filter by selecting different dimensions, measures, and filters.
+* Profile API Flows include: +
+** <<Profile - List Metadata API>> +
+Shows how to retrieve the metadata for all data model objects.
+** <<Profile - Get Metadata API>> +
+Shows how to retrieve the metadata for the requested data model object.
+** <<Profile - Search Records API>> +
+Shows how to retrieve data model object records based on search filters.
+** <<Profile - Search Records By Id API>> +
```
Comment (DonaldBough) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]: Looks like this should be: Profile - Search Records By ID API. The link didn't work in my IntelliJ adoc previewer.

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,1163 @@
...
+Shows how to retrieve data model object records based on indexes and search filters.
+** <<Profile - Search Records With Child Records API>> +
+Shows how to retrieve data model object records and child object records based on indexes and search filters.
+** <<Profile - Search Records With Insight API>> +
+Shows how to retrieve data model object records and a computed view based on indexes and search filters.
+** <<Data Action Webhook>> +
```
Comment (DonaldBough) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc:47]: Data Action Webhook isn't really grouped with the "Profile API Flows". I'd suggest keeping it by itself without a sublist or creating a sublist just for it with something like "Data Action API Flows".

**Review by DonaldBough — DISMISSED** [review]

File: salesforce-cdp/1.1/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+* Anypoint Studio version 7.3.x or later
+
+== Common Use Cases For the Connector
+
+These are some common use cases for Salesforce CDP Connector:
+
+* Stream data via insert and delete object endpoints of the Ingestion API Connector
+* Create a bulk job, upload job data, close or abort a job, get a job status, and delete a job as part of the Bulk Ingestion interaction pattern of the Ingestion API Connector
+* Query CDP objects
+* Query Salesforce CDP data lake across data model, lake, unified, and linked objects
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/index.adoc:34]:
```suggestion
* Query Salesforce the CDP data lake across the data model, data lake, unified objects, and linked objects
```

Comment (dhirsch1) [reply salesforce-cdp/1.1/modules/ROOT/pages/index.adoc:34]: I think "unified" refers to objects, but am not positive.  Also, it seems like the objects are part of the data lake, so you might need to say: ."...across the data model and data lake, and across unified and linked data lake objects."

File: salesforce-cdp/1.1/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+* Query calculated insights with selected dimensions and measures, with optional filters
+* Lookup and search customer profile information
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/index.adoc]: "Look up" when used as a verb

File: salesforce-cdp/1.1/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+== Common Use Cases For the Connector
+
+These are some common use cases for Salesforce CDP Connector:
+
+* Stream data via insert and delete object endpoints of the Ingestion API Connector
+* Create a bulk job, upload job data, close or abort a job, get a job status, and delete a job as part of the Bulk Ingestion interaction pattern of the Ingestion API Connector
+* Query CDP objects
+* Query Salesforce CDP data lake across data model, lake, unified, and linked objects
+* Query calculated insights with selected dimensions and measures, with optional filters
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/index.adoc]: Suggestion: Query calculated insights with selected dimensions, selected measures, and optional filters

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 6]
```diff
@@ -0,0 +1,1164 @@
...
+. Save the project.
+. <<Create Job - Bulk API, Create a job>> and copy the resulting job ID.
+. Use the job you copied to close the job (see <<Close Job - Bulk API>> flow).
+. Test the flow by sending a `DELETE` to
+`localhost:8081/jobs/delete/{JOB_ID}`, using the job ID that you copied earlier.
+
+
+== Insights - List Metadata API
+
+This Mule flow shows how to get the metadata including the dimensions and filters for all calculated insights.
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]:
```suggestion
This Mule flow shows how to get the metadata, including the dimensions and filters for all calculated insights.
```

Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]: There are other occurrences of the text above  text to look for.

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 7]
```diff
@@ -0,0 +1,1164 @@
...
+
+This example uses the following operations:
+
+* *HTTP Listener* +
+Accepts HTTP GET requests.
+* *Insights - List Metadata*
++
+** Authenticates based on your choice of either JWT or username and password.
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]:
```suggestion
** Authenticates based on your choice of either a JWT or a username and password.
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 8]
```diff
@@ -0,0 +1,1164 @@
...
+=== Steps for Running This Example
+
+. Verify that your connector is configured.
+. Save the project.
+. Test the flow by sending a `GET` to
+`localhost:8081/insight/metadata`.
+
+== Insights - Get Metadata API
+
+This Mule flow shows how to retrieve the metadata including the dimensions and filters for a calculated insight. Refer to the https://developer.salesforce.com/docs/atlas.en-us.c360a_api.meta/c360a_api/c360a_api_insights_meta_ci_name.htm[Salesforce CDP API Developer Guide] for usage and further details.
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]:
```suggestion
This Mule flow shows how to retrieve the metadata, including the dimensions and filters for a calculated insight. Refer to the https://developer.salesforce.com/docs/atlas.en-us.c360a_api.meta/c360a_api/c360a_api_insights_meta_ci_name.htm[Salesforce CDP API Developer Guide] for usage and further details.
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc (hunk 7 above)
Comment (dhirsch1) [reply salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]: There are other occurrences of the above text.

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 9]
```diff
@@ -0,0 +1,1164 @@
...
+=== Steps for Running This Example
+
+. Verify that your connector is configured.
+. Save the project.
+. Test the flow by sending a `GET` to
+`localhost:8081/insight/calculated-insights/{CALCULATED_INSIGHT_NAME}`.
+
+== Profile - List Metadata API
+
+This Mule flow shows how to retrieve the list of data model objects, their fields, and category. Refer to the https://developer.salesforce.com/docs/atlas.en-us.c360a_api.meta/c360a_api/c360a_api_profile_meta.htm[Salesforce CDP API Developer Guide] for usage and further details.
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]:
```suggestion
This Mule flow shows how to retrieve the list of data model objects and their fields and category. Refer to the https://developer.salesforce.com/docs/atlas.en-us.c360a_api.meta/c360a_api/c360a_api_profile_meta.htm[Salesforce CDP API Developer Guide] for usage and further details.
```

Comment (dhirsch1) [reply salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]: Ditto with other occurrences of this text.

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 10]
```diff
@@ -0,0 +1,1164 @@
...
+
+. Verify that your connector is configured.
+. Save the project.
+. Test the flow by sending a `GET` to
+`localhost:8081/profile/{DATA_MODEL_NAME}/{Id}/calculated-insights/{CALCULATED_INSIGHT_NAME}`.
+
+
+== Data Action Webhook
+
+This Mule flow shows how to create a webhook that will log its payload when triggered by a Data Action. Refer to the https://help.salesforce.com/s/articleView?language=en_US&type=5&id=sf.c360_a_data_actions.htm[Salesforce Data Action docs].
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]:
```suggestion
This Mule flow shows how to create a webhook that logs its payload when triggered by a Data Action. Refer to the https://help.salesforce.com/s/articleView?language=en_US&type=5&id=sf.c360_a_data_actions.htm[Salesforce Data Action docs].
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 11]
```diff
@@ -0,0 +1,1164 @@
...
+		<logger level="INFO" doc:name="Log Payload" doc:id="716538d3-7d06-4e9c-a4f7-b0d350b4dafe" message="#[output application/json --- payload]"/>
+	</flow>
+</mule>
+----
+
+=== Steps for Running This Example
+
+. Verify you have https://help.salesforce.com/s/articleView?id=sf.c360_a_create_streaming_insight.htm&type=5[created a Streaming Insight].
+. Verify you https://help.salesforce.com/s/articleView?id=sf.c360_a_data_actions.htm&type=5[created a Data Action Target] with an Action Target Type of `Webhook`.
+. Verify you https://help.salesforce.com/s/articleView?id=sf.c360_a_data_actions.htm&type=5[created a Data Action] running on a condition against your streaming Calculated Insight, that will use the Data Action Target.
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc]: This text is confusing. Maybe:
Verify that you https://help.salesforce.com/s/articleView?id=sf.c360_a_data_actions.htm&type=5[created a Data Action that uses the Data Action Target and that runs on a condition against your streaming Calculated Insight.

(I am not sure if my rewrite makes sense in the context of the product.) What does it mean to "run on a condition"?

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc [hunk 12]
```diff
@@ -0,0 +1,2035 @@
...
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| TLS_Config a| <<Tls>> |  Reference to a TLS configuration element. Enables HTTPS for this configuration. |  |
+| Protocol a| Enumeration, one of:
+
+** HTTP
+** HTTPS |  Protocol to use for communication. When using HTTPS, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as protocol, the user must configure the keystore in the `tls:context` child element of this listener configuration. |  HTTP |
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc]:
```suggestion
** HTTPS |  Protocol to use for communication. When using HTTPS, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, the user must configure the keystore in the `tls:context` child element of this listener configuration. |  HTTP |
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc [hunk 13]
```diff
@@ -0,0 +1,2035 @@
...
+| offset a| Number |  Number of rows to skip before starting to return. |  |
+| filters a| String |  This parameter is added to a select query to select only results of a specific type. |  |
+| orderby a| String |  Comma-separated values to sort the result-set in ascending or descending order. |  |
+| timeGranularity a| Enumeration, one of:
+
+** HOUR
+** DAY
+** MONTH
+** QUARTER
+** YEAR |  User specified granularity if *ci* has one or more time dimensions available. |  |
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc]:
```suggestion
** YEAR |  User-specified granularity if *ci* has one or more time dimensions available. |  |
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc [hunk 14]
```diff
@@ -0,0 +1,2035 @@
...
+| dataModelName a| String |  Data model object API name. |  | x
+| id a| String |  Value of an indexed column. |  | x
+| ci_name a| String |  Calculated insight name. |  | x
+| searchKey a| String |  Index name if an object's ID is different from the primary key. |  |
+| dimensions a| String |  Comma-separated name of the dimensions to be a part of the result from the computed view. By default, this parameter includes all the dimensions. |  |
+| measures a| String |  Comma-separated name of the measures to be a part of the result from the computed view. By default, this parameter includes all the measures. |  |
+| limit a| Number |  Number of items to return. |  |
+| filters a| String |  Filters are applied only on the parent object. |  |
+| offset a| Number |  Number of rows to skip before starting to return. |  |
+| orderby a| String |  Comma-separated values to sort the result-set in ascending or descending order. |  |
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc]:
```suggestion
| orderby a| String |  Comma-separated values to sort the result set in ascending or descending order. |  |
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc [hunk 15]
```diff
@@ -0,0 +1,2035 @@
...
+| timeGranularity a| Enumeration, one of:
+
+** HOUR
+** DAY
+** MONTH
+** QUARTER
+** YEAR |  User specified granularity if *ci* has one or more time dimensions available. |  |
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc]: Global: user specified >  user-specified

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc [hunk 16]
```diff
@@ -0,0 +1,2035 @@
...
+Returns data model objects based on search indexes and filters. Use this API call to retrieve the object record based on the primary key or secondary keys. Returns an empty list when records are not found. This operation makes an HTTP GET request to the /profile/{dataModelName}/{id} endpoint.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| dataModelName a| String |  Data model object API name. |  | x
+| Id a| String |  Value of the primary or secondary key field. Defaults to primary key when not specified. |  | x
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc]:
```suggestion
| Id a| String |  Value of the primary or secondary key field. Defaults to the primary key when not specified. |  | x
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc [hunk 17]
```diff
@@ -0,0 +1,2035 @@
...
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| dataModelName a| String |  Data model object API name. |  | x
+| id a| String |  Value of an indexed column. |  | x
+| childDataModelName a| String |  Child data model object API name. |  | x
+| searchKey a| String |  Index name if an object's ID is different from the primary key. |  |
+| fields a| String |  Comma-separated name of the fields of the result from the child entity. If unspecified, the first ten alphabetically sorted fields are given. |  |
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc]:
```suggestion
| fields a| String |  Comma-separated name of the fields of the result from the child entity. If unspecified, the first ten alphabetically-sorted fields are given. |  |
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc [hunk 18]
```diff
@@ -0,0 +1,2035 @@
...
+* SDC:TOO_MANY_REQUESTS
+* SDC:UNAUTHORIZED
+* SDC:UNSUPPORTED_MEDIA_TYPE
+
+
+[[GetParentWithFilters]]
+== Profile - Search Records
+`<sdc:get-parent-with-filters>`
+
+Returns data model objects based on search filters. Use this API call to retrieve the object records after applying the selected filter(s). This operation makes an HTTP GET request to the /profile/{dataModelName} endpoint.
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-reference.adoc]:
```suggestion
Returns data model objects based on search filters. Use this API call to retrieve the object records after applying the selected filters. This operation makes an HTTP GET request to the /profile/{dataModelName} endpoint.
```

File: salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc [hunk 19]
```diff
@@ -0,0 +1,1164 @@
...
+
+* *HTTP Listener* +
+Accepts data from HTTP POST requests with the source and object name as the URI parameters.
+* *Logger* +
+Shows the HTTP request received from the *Listener*, then later shows the HTTP response from the *Insert Objects* operation.
+* *Streaming Insert*
++
+** Authenticates based on your choice of either JWT or username and password.
+** Receives the source name, object name, and payload.
+** Calls the Salesforce API to insert the payload to that object's endpoint.
```
Comment (dhirsch1) [inline salesforce-cdp/1.1/modules/ROOT/pages/salesforce-cdp-connector-examples.adoc:65]: This wording isn't quite right, but it's up to you whether you want to change it here and elsewhere. The meaning is clear.

Suggestion: Requests that the Salesforce API insert...

**Review by dhirsch1 — CHANGES_REQUESTED** [review]

File: salesforce-cdp/1.1/modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (sathya0) [reply salesforce-cdp/1.1/modules/ROOT/pages/index.adoc:34]: It is referring to data model objects, lake objects, unified objects, and linked objects

**Review by dhirsch1 — APPROVED** [review]

**Review by dhirsch1 — DISMISSED** [review]

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**